tests/unit/amo/components/TestCollection.js

```diff
@@ -1315,53 +1281,7 @@ describe(__filename, () => {
     const root = renderComponent({
       editing: true,
       errorHandler,
-      location: fakeRouterLocation({ query: { page } }),
-      store,
-    });
-
-    fakeDispatch.resetHistory();
-
-    // This simulates the user clicking the "Save" button on the
-    // EditableCollectionAddon component's comment form.
-    root.instance().saveNote(addonId, errorHandler, notes);
-    sinon.assert.callCount(fakeDispatch, 1);
-    sinon.assert.calledWith(
-      fakeDispatch,
-      updateCollectionAddon({
-        addonId,
-        errorHandlerId: errorHandler.id,
-        notes,
-        page,
-        slug: collectionDetail.slug,
-        username: collectionDetail.author.username,
-      }),
-    );
-  });
-
-  it('dispatches updateCollectionAddon when saveNote is called without a page defined', () => {
```

Comment:
As mentioned above, this test case was removed because there's no need to test the functionality `without a page defined`, because that no longer applies to this component.